package.json

```diff
@@ -9,17 +9,20 @@
         "services/*"
     ],
     "devDependencies": {
-        "@dhis2/cli-style": "^3.0.1",
+        "@dhis2/cli-style": "~3.0.0",
```

Comment:
**NOTE** I had to change this to a ~ range (so <3.1.0) because eslint was introduced in `cli-style@3.1.0` and doesn't play nice with Typescript.  It would be nice to [support Typescript with our ESLint config](https://43081j.com/2019/02/using-eslint-with-typescript)

Reply:
Working on supporting external ESLint plugins in: https://github.com/dhis2/cli-style/pull/72

Reply:
You can upgrade to 4.x of cli-style and get support for extended ESLint configs.